src/main/java/smile/identity/core/models/PartnerParams.java

```diff
@@ -12,13 +12,13 @@ public class PartnerParams {
     @Json(name = "job_type")
     JobType jobType;
 
-    @Json(name = "optional_info")
-    Map<String, Object> optionalInfo;
-
     @Json(name = "user_id")
     String userId;
 
     @Json(name = "job_id")
     String jobId;
 
+    @Json(name = "optional_info")
```

Comment:
What's the reasoning behind changing the signature? I don't necessarily disagree with it if it has a purpose!

Reply:
I changed it so that the "less important" parameter would come last in the constructor. Not really  _that_ important of a change, just thought it would read better. @beastawakens

Reply:
I'd agree that that is a better pattern to follow for sure 😊 I guess it's just a breaking change for anyone updating to the newer version of the library, but it's not that drastic to just change the order 😄